.meta/_partials/fields/_tls_connector_options.toml.erb

```diff
@@ -10,7 +10,7 @@ description = "Configures the TLS options for connections from this sink."
 [<%= namespace %>.tls.children.enabled]
 type = "bool"
 common = true
-default = false
+default = <%= can_enable_default %>
```

Comment:
If I'm reading this right, I'd just name this `enabled_default`. Either that, or make `can_enable` work as a tristate, where unset drops the `enabled` section, and `true` or `false` set the default.

Reply:
I wanted use `can_enable_*` format of the name to make it clearer that it's an field under `can_enabled`, but it seems that `can` is getting misinterpreted. 

Tristate would differ from all of the other settings in a sneaky way, as in:
```
  can_enable: false,
  can_verify_certificate: false,
```
would mean `can_verify_certificate` section isn't enabled, but `can_enable` section is enabled, and yet both of them have `false` value.

`enabled_default` is fine so let's go with that.